src/controllers/legacy/cmc.ts

```diff
@@ -1,6 +1,6 @@
 import { Router } from 'express';
 
-import { LONG_CACHE_TTL } from 'config/constants';
+import { DECIMAL_PLACES, LONG_CACHE_TTL } from 'config/constants';
```

Comment:
```suggestion
import { LONG_CACHE_TTL } from 'config/constants';
```
`DECIMAL_PLACES` is not used.